### DISCUSSION TITLE
Default Stage size when opening editor

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6559] [Ideas]
During my lessons I must open several Scratch editors in order to show all the projects relatated to the features that I'm explaining. To make the Scratch blocks clearly visible on the screen of the overhead projector, I always use the small Stage. So, every time I open a new editor I have to resize the Stage.

It would be useful for me having the possibility to set up a default editor size, choosen among the small/large/no stage alternatives.

**Comment by WorldLanguages** [comment]
Created https://github.com/ScratchAddons/ScratchAddons/issues/6562
